example/manifests/observatorium.yaml

```diff
@@ -46,9 +46,6 @@ spec:
       keyKey: key.pem
       secretName: observatorium-xyz-tls
     version: master-2020-11-02-v0.1.1-192-ge324057
-  apiQuery:
-    image: quay.io/thanos/thanos:v0.17.1
```

Comment:
Why do we need to remove this?

Reply:
When applying the example CR I get the following error:

```
error: error validating "https://raw.githubusercontent.com/observatorium/operator/master/example/manifests/observatorium.yaml": error validating data: [ValidationError(Observatorium.spec): unknown field "apiQuery" in io.observatorium.core.v1alpha1.Observatorium.spec, ValidationError(Observatorium.spec.securityContext): unknown field "fsGroup" in io.observatorium.core.v1alpha1.Observatorium.spec.securityContext]; if you choose to ignore these errors, turn validation off with --validate=false
```

Looking at the spec, I don't see `apiQuery` either.
https://github.com/observatorium/operator/blob/master/api/v1alpha1/observatorium_types.go#L26-L62

Reply:
Was it supposed to be `queryFrontend`?

https://github.com/observatorium/operator/blob/08f589b69dbd6a0072f7af8f5225c93c905c1fb2/api/v1alpha1/observatorium_types.go#L38

Reply:
This seems correct to me, according to https://github.com/observatorium/operator/pull/9 it is not being used anymore and has been removed from the spec.